components/lifter.py

```diff
@@ -98,16 +113,23 @@ def move(self, input_setpos):
             input_setpos (int): Encoder position to move lift to in m.
         """
         self.set_pos = input_setpos
-        self.motor.set(self.motor.ControlMode.MotionMagic, self.metres_to_counts(self.set_pos))
+        if self.set_pos - self.get_pos() < 0:
+            self.motor.configMotionAcceleration(self.DOWNWARD_ACCELERATION, timeoutMs=10)
+        else:
+            self.motor.configMotionAcceleration(self.UPWARD_ACCELERATION, timeoutMs=10)
```

Comment:
So we probably want these to have no timeout - i.e. fire-and-forget. It shouldn't be a huge problem if they aren't initially set correctly, as long as they eventually are. (Also, it looks like these are both the same currently anyway.)